.env.example

```diff
@@ -137,3 +137,10 @@ MAX_EMAILS_PER_USER = 10
 
 # Max training report size in bytes
 MAX_TRAINING_REPORT_UPLOAD_SIZE = 1048576
+
+# Session settings
```

Comment:
worth commenting what the unit of measurement is on the expiry (I assume seconds)